src/v0/destinations/fb_custom_audience/util.js

```diff
@@ -49,14 +49,13 @@ const batchingWithPayloadSize = (payload) => {
 };
 
 const getSchemaForEventMappedToDest = (message) => {
-  const mappedSchema = get(message, 'context.destinationFields');
+  const mappedSchema = get(message, 'fields');
```

Comment:
we shouldn't use get when we know the exact path.

message?.fields should be preferred